azure-ts-dynamicresource/index.ts

```diff
@@ -54,7 +55,7 @@ const cdnEndpoint = new azure.cdn.Endpoint("cdnEndpoint", {
      * For example, the URL for this CDN endpoint when it is created 
      * would be `my-cdn-endpoint.azureedge.net`.
      */
-    name: "my-cdn-endpoint",
+    name: `my-cdn-endpoint-${cdnSuffix}`,
```

Comment:
Why is the suffix needed?

Reply:
This test is *all* sorts of broken - I am trying to do something to get it working - I will remove this again